src/solver.jl

```diff
@@ -10,8 +10,10 @@ type PODNonlinearModel <: MathProgBase.AbstractNonlinearModel
     timeout::Float64                                            # Time limit for algorithm (in seconds)
     maxiter::Int                                                # Target Maximum Iterations
     relgap::Float64                                             # Relative optimality gap termination condition
+    gapref::AbstractString                                      # Relative gap reference point
```

Comment:
This option seems unneccessary - we can do what Gurobi or BARON does and stick with it

Reply:
I think BARON is consistent with CPLEX but I cannot what rule Gurobi uses. SCIP would do whichever is the smallest. It seems that each solver has its own rule. Should we let the user choose so that the solver can terminate properly for benchmarking?

Reply:
No lets just stick with what BARON or CPLEX does. Forget Gurobi or CPLEX for now.